docs/book/src/topics/vpc/creating-a-cluster.md

```diff
@@ -1 +1,163 @@
 # Creating a cluster
+
+### Provision local boostrap management cluster:
```

Comment:
I'm wondering if this generic instruction of deploying the bootstrap cluster can be moved to the getting started section, even information from PowerVS also can be moved there.

Reply:
@Prajyot-Parab ^^

Reply:
Yes, this makes sense. I shifted the generic content to getting-started section. PTAL.